substrate-archive-backend/Cargo.toml

```diff
@@ -22,7 +22,7 @@ parity-util-mem = "0.8"
 
 # Substrate
 sc-client-api = { git = "https://github.com/paritytech/substrate", branch = "master" }
-sc-executor = { git = "https://github.com/paritytech/substrate", branch = "master" }
+sc-executor = { git = "https://github.com/paritytech/substrate", branch = "master", features = ["wasmtime"] }
```

Comment:
`wasmtime` only works for `x64` last I checked,

[substrate does this for example](https://github.com/paritytech/substrate/blob/master/bin/node/cli/Cargo.toml#L108-#L112)

You don't have to fix it in this PR but we should file an issue because most likely someone is going to run into this problem.

Reply:
Didn't know this actually! By default we run in interpreted which shouldn't use wasmtime, but this is good to know. Created #209 and #210 for this